test/integ/quote-classic.test.ts

```diff
@@ -70,6 +70,7 @@ const axios = axiosStatic.create();
 const axiosConfig: AxiosRequestConfig<any> = {
   headers: {
     ...(process.env.URA_INTERNAL_API_KEY && { 'x-api-key': process.env.URA_INTERNAL_API_KEY }),
+    ...(process.env.FORCE_PORTION_SECRET && { 'X-UNISWAP-FORCE-PORTION-SECRET': process.env.FORCE_PORTION_SECRET}),
```

Comment:
also need to add the test header in [test/integ/quote-gouda.test.ts](https://github.com/Uniswap/unified-routing-api/blob/main/test/integ/quote-gouda.test.ts)